tests/e2e_logging.py

```diff
@@ -674,6 +657,49 @@ def test_historical_receipts(network, args):
     return network
 
 
+def get_all_entries(client, target_id, from_seqno=None, to_seqno=None, timeout=5):
+    LOG.info(
+        f"Getting historical entries{f' from {from_seqno}' if from_seqno is not None else ''}{f' to {to_seqno}' if to_seqno is not None else ''} for id {target_id}"
+    )
+    logs = []
+
+    start_time = time.time()
+    end_time = start_time + timeout
+    entries = []
+    path = f"/app/log/private/historical/range?id={target_id}"
+    if from_seqno is not None:
+        path += f"&from_seqno={from_seqno}"
+    if to_seqno is not None:
+        path += f"&to_seqno={to_seqno}"
+    while time.time() < end_time:
+        r = client.get(path, log_capture=logs)
```

Comment:
Can we make this use JWT, so it's closer to what we generally are trying to replicate here?

Reply:
Yes. Since we already take the client as an argument, I've added a `common_headers` arg to `CCFClient` to have add the same headers to every request (ie - to use bearer auth), so we can create a JWT-auth'd client that's indistinguishable from a cert-authed at calling points like this.